tasks/components/layout/WardDetails.tsx

```diff
@@ -0,0 +1,129 @@
+import { tw, tx } from '@helpwave/common/twind'
+import type { Languages } from '@helpwave/common/hooks/useLanguage'
+import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
+import { useState } from 'react'
+import { ColumnTitle } from '../ColumnTitle'
+import { Button } from '../Button'
+import { RoomList } from '../RoomList'
+import { WardForm } from '../WardForm'
+
+type WardDetailTranslation = {
+  updateWard: string,
+  updateWardSubtitle: string,
+  createWard: string,
+  createWardSubtitle: string,
+  dangerZone: string,
+  dangerZoneText: string,
+  deleteConfirmText: string,
+  deleteWard: string,
+  create: string,
+  update: string
+}
+
+const defaultWardDetailTranslations: Record<Languages, WardDetailTranslation> = {
+  en: {
+    updateWard: 'Update Ward',
+    updateWardSubtitle: 'Here you can update details about the ward such as rooms',
+    createWard: 'Create Ward',
+    createWardSubtitle: 'Here you set the details of the new ward',
+    dangerZone: 'Danger Zone',
+    dangerZoneText: 'Deleting the ward is a permanent action and cannot be undone. Be careful!',
+    deleteConfirmText: 'Do you really want to delete this ward?',
+    deleteWard: 'Delete ward',
+    create: 'Create',
+    update: 'Update'
+  },
+  de: {
+    updateWard: 'Station ändern',
+    updateWardSubtitle: 'Hier kannst du die Details der Station ändern.',
+    createWard: 'Station erstellen',
+    createWardSubtitle: 'Hier setzt du die Details der Station.',
+    dangerZone: 'Gefahren Zone',
+    dangerZoneText: 'Das Löschen einer Station is permanent und kann nicht rückgängig gemacht werden. Vorsicht!',
+    deleteConfirmText: 'Wollen Sie wirklich diese Station löschen?',
+    deleteWard: 'Station Löschen',
+    create: 'Erstellen',
+    update: 'Ändern'
+  }
+}
+
+type Room = {
+  bedCount: number,
+  name: string
+}
+
+type WardDTO = {
+  id: string,
+  name: string,
+  rooms: Room[],
+  unscheduled: number,
+  inProgress: number,
+  done: number
+}
```

Comment:
I saw this concept once in a while...
Can't we just use the DTO defined in the mutations? Whats the reason for this?

Reply:
You mean the multiple type definitions, right? It's to make the components more independent, thereby increasing the reusability. It also allows each component to only require the subset of attribute it needs. We can change that or discuss it, I don't mind importing the definitions

Reply:
Okay, that's resolved. I'm facing this in #161
So I'll probably fix it there.